src/wallet/components/WalletDropdownFundLink.tsx

```diff
@@ -0,0 +1,81 @@
+import { isValidElement, useMemo } from 'react';
+import { FundWalletSvg } from '../../internal/svg/fundWallet';
+import { cn, pressable, text as themeText } from '../../styles/theme';
+import { version } from '../../version';
+import type { WalletDropdownFundLinkReact } from '../types';
+
+export function WalletDropdownFundLink({
+  className,
+  icon,
+  rel,
+  openIn = 'tab',
+  target,
+  text = 'Fund wallet',
+  windowSize = 'm',
+}: WalletDropdownFundLinkReact) {
+  const iconSvg = useMemo(() => {
+    if (icon === undefined) {
+      return FundWalletSvg;
+    }
+    if (isValidElement(icon)) {
+      return icon;
+    }
+  }, [icon]);
+
+  const currentURL = window.location.href;
+  const tabName = document.title;
+  const fundingUrl = `http://keys.coinbase.com/funding?dappName=${tabName}&dappUrl=${currentURL}&onchainkit=${version}`;
+  const windowSizes: Record<
+    's' | 'm' | 'l',
+    {
+      width: number;
+      height: number;
+    }
+  > = {
+    s: { width: 400, height: 500 },
```

Comment:
Who is deciding those sizes? are those standardize anywhere?

Reply:
nope - just values I think look good from testing on a variety of screen sizes. changed to viewport units instead

Reply:
ok if that is the case, would love to use different size wording, like `small`, `medium`, `large`.

Just dropping s, m, l, I am worry to start some bad practice. As this is the first time we do this kind of thing.